_rules/7d6734.md

```diff
@@ -0,0 +1,207 @@
+</svg>
+```
+
+#### Failed example 3
+
+The `<svg>` element is included in the accessibility tree with an explicit role of `img`, has visible content rendered by the `<circle>` element, but has no content in the `<title>` element that is being used for the accessible name.
+ 
+```html
+<svg xmlns="http://www.w3.org/2000/svg" role="img" width="100" height="100">
+	<title></title>
+  	<circle role="none" cx="50" cy="50" r="40" fill="yellow"></circle>
+</svg>
+```
+
+#### Failed example 4
+
+The `<circle>` elements have visible content, are included in the accessibility tree with the explicit role `graphics-symbol` but they have no have accessible names.
+
+```html
+<svg xmlns="http://www.w3.org/2000/svg" role="presentation">
+	<circle role="graphics-symbol" cx="50" cy="50" r="40" stroke="green" stroke-width="4" fill="yellow"></circle>
+	<circle role="graphics-symbol" cx="150" cy="50" r="40" stroke="green" stroke-width="4" fill="yellow"></circle>
+	<circle role="graphics-symbol" cx="250" cy="50" r="40" stroke="green" stroke-width="4" fill="yellow"></circle>
+</svg>
+```
+
+#### Failed example 5
+ 
+The `<svg>` element has a role of `img`, has visible content rendered by the `<circle>` element, but no accessible name. The `title` attribute is not defined in the SVG namespace.
+
+```html
+<svg xmlns="http://www.w3.org/2000/svg" role="img" title="previous page">
+  	<circle cx="50" cy="50" r="40" fill="yellow"></circle>
+</svg>
+```
+
+#### Failed example 6
+ 
+The `<circle>` element has visible content, is included in the accessibility tree with a valid `tabindex` attribute, but has no accessible name.
+
+```html
+<svg xmlns="http://www.w3.org/2000/svg">
+  	<circle tabindex="0" cx="50" cy="50" r="40" fill="yellow"></circle>
+</svg>
+```
+
+### Inapplicable
+
+#### Inapplicable example 1
+
+The `svg` element does not render any visible content to the screen and is not included in the accessibility tree. 
+
+```html
+<svg xmlns="http://www.w3.org/2000/svg" width="100" height="100">
+</svg>
+```
+
+#### Inapplicable example 2
+
+The `<svg>` element has visible content rendered by the `<circle>` element but with no explicit role indicating its purpose it is not included in the accessibility tree.
+
+```html
+<svg xmlns="http://www.w3.org/2000/svg" width="100" height="100">
+  	<circle cx="50" cy="50" r="40" fill="yellow"></circle>
+</svg>
+```
+
+#### Inapplicable example 3
+
+The `svg` element has visible content rendered by the `<circle>` element, has an accessible name but the `aria-hidden` attribute removed the element from the accessibility tree.
+
+```html
+<svg xmlns="http://www.w3.org/2000/svg" role="img" aria-label="A yellow circle" aria-hidden="true">
+	<circle cx="50" cy="50" r="40" fill="yellow"></circle>
+</svg>
+```
+
+#### Inapplicable example 4
+
+The `svg` element does not render any visible content.
+
+```html
+<svg xmlns="http://www.w3.org/2000/svg" role="img" aria-label="A yellow circle" aria-hidden="true">
```

Comment:
I think we should remove the `aria-hidden` attribute from this example to make it really different from the previous one.

Reply:
Agreed